include/z64player.h

```diff
@@ -454,24 +690,52 @@ typedef struct struct_80122744_arg1 {
 #define PLAYER_STATE3_80000000   (1 << 31)
 
 
+#define PLAYER_GET_INITMODE(thisx) (((thisx)->params & 0xF00) >> 8)
+
+typedef enum {
+    /* 0x0 */ PLAYER_INITMODE_0,
+    /* 0x1 */ PLAYER_INITMODE_1,
```

Comment:
This is the mastersword one, right? Since that one is unused here I guess we should leave naming it to OoT.
I could leave a comment about it here tho